SPECS/envoy/envoy.spec

```diff
@@ -266,8 +267,13 @@ fdupes %{buildroot}%{src_install_dir}
 %{src_install_dir}
 
 %changelog
+* Tue Sep 14 2021 Henry Li <lihl@microsoft.com> - 1.14.4-4
+- Add patch to use newer version of bazel
+- Update patch to use new version of external dependencies
+- Update vendor source and file name
+
 * Tue Jun 15 2021 Henry Li <lihl@microsoft.com> - 1.14.4-3.4
-- Initial CBL-Mariner import from OpenSUSE Tumbleweed (license: same as "License" tag).
+- Initial CBL-Mariner import from OpenSUSE Tumbleweed
```

Comment:
Why was this entry changed? We need to mention under what license the spec is available.